login-module/src/index.js

```diff
@@ -13,10 +13,14 @@ window.GraphAcademyLogin = class GraphAcademyLogin {
 			console.log(`required params missing - one of ${constants.REQUIRED_OPTIONS.join(', ')}`);
 			return;
 		}
-
-		if (!Auth0Lock || typeof Auth0Lock !== 'function') return;
-		this.lock = new Auth0Lock('hoNo6B00ckfAoFVzPTqzgBIJHFHDnHYu', 'login.neo4j.com', constants.auth0Options);
-
+		this.webAuth = new WebAuth({
+			clientID: 'hoNo6B00ckfAoFVzPTqzgBIJHFHDnHYu',
+			domain: 'login.neo4j.com',
+			redirectUri: `${window.location.origin}/accounts/login`,
+			audience: 'neo4j://accountinfo/',
+			scope: 'read:account-info openid email profile user_metadata',
```

Comment:
One small change here, I've removed `write:account-info` because I don't think we need this scope right?